lib/reporters/bower/index.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs          = require( 'fs' );
+var path        = require( 'path' );
+var creditUtil  = require( '../../../util/credit' );
+var packageUtil = require( '../../../util/package' );
+
+
+/**
+* Read project root and evaluate dependency credits for bower modules
+*
+* @param  {String} projectPath      absolute path to project root
+*
+* @return {Array}                   list of credits
+*/
+function getBowerCredits( projectPath ) {
+  var credits = [];
+
+  var depPath = path.join( projectPath, 'bower_components' );
+
+  // projects without dependencies won't have
+  // a bower_components folder
+  try {
+    var deps = fs.readdirSync( depPath );
+  } catch( error ) {
+    return credits;
+  }
+
+  deps.forEach( function( name ) {
+    var bowerJsonPath = path.join( depPath, name, 'bower.json' );
+
+    // it might possible that some modules are missing
+    // the bower.json file
+    try {
+      var stats = fs.statSync( bowerJsonPath );
+    } catch ( error ) {
+      return credits;
+    }
+
```

Comment:
I don't get this part.

I assume this your check for an existant `bower.json` file?

Things to note here:
- `path.join( depPath, name, 'bower.json' )` is done twice
- `stats` is declared but not used

May I make a suggestion? Why not trying to read `bower.json` and if that fails, journes is over?

```
try {
  var bowerJson = require( bowerJsonPath );
} catch( error ) {
  return credits;
}
```